lib/opensusebasetest.pm

```diff
@@ -5,10 +5,19 @@ use testapi;
 
 # Base class for all openSUSE tests
 
+
+sub clear_console() {
+    # 'ctrl-l' does not get queued up in buffer. If this happens to fast, the
+    # screen would not be cleared
+    script_run "clear";
```

Comment:
no script_run - this adds another wait_idle after the call. clear is simply not a script, so we don't need for the system to finish idling, especially if the clear is followed by assert_screens that wait anyway